swig/array_nocpy.i

```diff
@@ -0,0 +1,295 @@
+
```

Comment:
I would like to add more bindings for other languages later and it is also handy to develop java bindings on desktop :)